scripts/watch_odoc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+inotifywait -qmr -e modify,create,delete ./_build/default/_doc/_html | xargs -n1 -I{} "./scripts/sync_odoc.sh"
```

Comment:
Can we add a platform detection if/else to choose if we use fswatch or inotifywait?
Because I don't have the latter on Mac